docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
+  nhais:
+     build:
+       context: ../
```

Comment:
the context and dockerfile properties in docker-compose will also need to be fixed